### PR TITLE
Changes the genetics XRAY to night vision

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
Removes the XRAY block from the game and replaces it with night vision.
Xray is still achievable with the implant and other ways. DNA vault and the wish granter. 
Vampires and wizards still have their Xray. I just removed the block so genetics can't get it and replaced it with a night vision block.

Currently antags have 1 play style if xray is found. Loud. Can't go be stealthy if security and maybe more crewmembers can see you. So what do you do? You perma kill the ones having xray since that way you survive. This is just plain boring.

Why not remove the science xray vision? That one is hard to make and you need atleast 1 other person to implant you. Thus making it more balanced.

Fix points: #9616 #9623 #9625

**Changelog:**
:cl: Farie82
add: Night vision genetics block
del: Xray vision genetics block
balance: Genetics now can't get xray vision anymore. This is replaced by night vision.
/:cl:

### CODE REVIEW

**Comment by davidchan13** [comment]
Does this affect Vampire and/or Wizard X-ray?

**Comment by farie82** [comment]
It leaves vamp xray and wiz xray in. I just removed the xray genetics block.
Why? Ever had a round as antag and sec had xray? Means you cant be stealthy. Means you have to perma kill them...

**Comment by tigercat2000** [comment]
I'm 100% against this. XRAY is one of the oldest powers in the game.

**Comment by tigercat2000** [comment]
Also, you need 3 fix tokens for a balance PR like this.

**Comment by TDSSS** [comment]
Something being old is not exactly an argument for why it is balanced, I don't think.

**Comment by tigercat2000** [comment]
@TDSSS It's making a point about how this being "unbalanced" is absolutely an act of perception and *perhaps* interlacing systems, because it sure as hell ain't a problem on it's own, and it's ridiculous to claim that it is.

If this is considered an issue because it's too easy to get, then I would think it's time for another genetics overhaul to deal with how much of a piece of garbage it is, or introducing harsher restrictions to the instability mechanic. If it's because of x-ray guns, well fine, remove them, go ahead, please do, they're kinda shit. But how do you figure you can find an actual issue with something that's been a game mechanic for so long?

**Comment by farie82** [comment]
I myself am fine with having Xray vission in the game. But currently you have 2 methods. Chance, genetics. Once found easy to distribute and with no cost to get it. Or science. Hard to get and hard to distribute. Need 2 people to implant one. One scientist to print and one doc to implant.

**Comment by tigercat2000** [comment]
As I said... if you believe the major issue here is not the power's existence, but how genetics impacts it, the answer is to actually work on rebalancing genetics (again), not removing the gene. We're currently unique in how our DNA system works - and considerably outdated. Everyone else has been fucking with genetics forever.

**Comment by Dave-TH** [comment]
@tigercat2000 Being one of the oldest powers doesn't mean anything. Re-balancing genetics is a major undertaking that would spawn a lot of discussion (And possibly remove x-ray anyhow! Notice how Goon's reworked genetics, also done by Keelin like our current genetics system, doesn't have x-ray). It's a better solution, but it would take a while to say the least. X-Ray is *the* most problematic gene of the set that we have right now, and its removal is 100% healthy for the game. There's not a single antag that doesn't get uncounterably screwed by genetics lucking their way into x-ray 30 minutes into a shift and handing it out to security like candy. While antags can *theoretically* also benefit from this, it almost certainly won't happen unless one of the two geneticists are antags as well.

**Comment by davidchan13** [comment]
Something being old to a role that has historically been circling the chopping block for years isn't really a point in its favor. Genetics has always been problematic with some powers being next to useless (Strong, Cold/Heat Resistance, even Regeneration) to exceptionally powerful and ripe for abuse (TK, Hulk, XRAY) with very little middle ground. Gating them behind RNGesus is not a good balance. X-ray in particular has always annoyed me with it's lack of a downside particularly relative to flash and welding protections still functioning perfectly fine on a power that literally lets you see through walls.

Is this a perfect solution? No, probably not. Genetics needs a rework and expansion of mid level powers with high level powers needing more work to unlock. Does this address concerns among the player base of Security, Command and other valid hunters having a good chance of access to a power that all but removes all elements of stealth from the game that can be acquired in the first 20 minutes of a round? Yes. It does that.

The implants still exist but require competent mining and science teams to produce. And use of the implants at least comes with the downside of EMP induced damage causing blindness, so if a sneaky gent happens to learn security is all getting implanted they can get an EMP kit or snag an Ion rifle to counter them.

**Comment by farie82** [comment]
Completely agree with David. It's not a perfect fix. But it's a fix to help make rounds more interesting for both parties. Security because they don't steamroll and (hopefully) face less powergaming antags. And for the antags who can actually rely on stealth now and don't need to resort to powergaming to get their objectives done once xray is found.

**Comment by Squirgenheimer** [comment]
- XRAY vision allows security players (and sometimes antags) to gather _practically all_ the information they could conceivably need.
- Players who get detected by XRAY vision have little to no warning concerning their detection. If, for example, an antag was seen by a security player who has normal vision, they would usually know that they were detected because they can also see them. 
This same issue occurs when a meson wearing engineer spots a cult base.

I think these are the essential components of why xray vision is a problem.
It's hard to see the merit in having to set high availability restrictions or other using balancing methods with a rather unfavorably viewed mechanic instead of introducing a different mechanic that is easier to balance because it isn't so powerful by itself (which xray is).

In other words: When choosing between moderate advantages coupled with moderate disadvantages and exceptional advantages coupled with exceptional disadvantages, it might be better to choose the former.

That's not to say I'm totally against keeping xray vision, but I would rather see it gone.

**Comment by Mitchs98** [comment]
I'm indifferent.
On one hand x-ray causes antags to powergame.
On the other it counters it quite nicely. You don't always have x-ray, but when you do and antags are going on 'lols murderbonespree time!' it's very lovely to have. I'm kind of leaning towards more not removing it just for this fact as it's a good hard counter to both antags that have it and antags that powergame and such.

If antags powergame all the time, people really need the tools to stop them. Though as Farie said if x-ray was removed...perhaps people would powergame less(though I doubt it, that is one of the only reasons I'm indifferent on keeping or removing it, but leaning towards being in favor of keeping.)

**Comment by ScratKnapp** [comment]
Does this PR also lower its genetic instability? Hardly fair to have such high instability for what would then only be night vision.

**Comment by farie82** [comment]
Does not lower it yet. I can lower it quite easily. Just need some feedback for that.

**Comment by variableundefined** [comment]
This PR has been rejected by a simple majority maintainer vote with a head also voting against it.

As per our 24-hours rules we will be waiting until 24 hours passed since the PR was opened before we close it.

**Comment by Necaladun** [comment]
I find it fascinating that something that has been around forever is being majorly seen as a problem now, rather than years ago. The optimist in me wants to say this is because the antag meta has gotten much more concerned with stealth rather than just outright murderboning. 

Rather than removing it from genetics outright, I'd much prefer to see it be nerfed - suggestions so far have been reduced view range, making people appear as skeletons (so knowing who they are is impossible), a hazy overlay to make it harder to see, etc.

This could also be tied in with an overhaul of how genetics works in general - a single injection, mass-produced, with basically no downsides, is a big part of how much of a problem this is, as all of the crew can be given injectors pretty easily.

**Comment by Cactus2388** [comment]
I'd rather it be removed now, because we all know that change will likely never happen. Shame that the decision of a handful can make a decision for ~hundreds of players.

**Comment by davidchan13** [comment]
@Necaladun It's been a problem forever, it's been overshadowed by things like hulk. At a glance it's impossible to tell if players have Xray, unlike hulk and tk, so it's harder to call out when it's been abused. 

@variableundefined @tigercat2000 Please explain to me how genetics xray is not a problem. It's superior in every way to the Xray implant, can be acquired sooner, requires no special materials to create or acquire, and can be (re)applied quicker without surgery. And genetics xray has no downside. The only 'counter' is to mutadone the person, which sets them back to regular vision. EMPing and implanted person can blind them, meaning the harder to acquire implant is riskier to use.

This all comes with the caveat that things like genetic Xray aren't purchasable by Nuke Ops.

**Comment by EvadableMoxie** [comment]
Arguing it should stay in the game simply because it's already in the game is a Fallacy. Specifically, The Tradition fallacy.  

It's irrelevant how long it's been in the game.  Why you should be arguing is _why_ it's been in the game all this time, and how it makes the game better.

Saying we shouldn't fix it now is another fallacy, the Nirvana fallacy, where you disregard improvement simply because it isn't perfect.  Surely if there is a massive overhaul of genetics down the line, changing night vision back to X-ray would be a simple thing as part of that larger update.

And, saying someone is wrong simply due to some of their arguments being fallacies is a fallacy in itself, so I am not saying that.  I'm asking for the people who support X-ray remaining as it is to provide logical and reasonable reasons.  What does X-ray add to the game?  How would removing it make the game worse? And thus far, no one has. 

If you can't honestly do that, then there really isn't an argument, is there?

**Comment by Dave-TH** [comment]
@Necaladun @variableundefined @tigercat2000 
Almost double the people (Literally double if you count the PR's owner, who didn't thumbs-up his own thing) believe that x-ray is problematic and that this PR is a good direction to head in.

You're offering us alternate solutions such as: Rework the whole genetics system, add a limited range for someone who is out of line-of-sight, swap a person's icon when they're out of line of sight, etc, but all of these would be nontrivial to implement (To say the least) and a pain to get merged. We also don't have anyone volunteering to do that work, which is fine and respectable. People have lives and commitments and there isn't anything wrong with that.

But we're closing a PR that would net a positive change for the game's health because there's a potential better change that no one has agreed to implement or even thoroughly thought out besides vague promises of "rework".

Please reconsider closing this PR.

**Comment by Bram961** [comment]
@Dave-TH I would like to add that speaking for myself and probably a few others, even if the genetics system is reworked or what have you, I include removing x-ray in that rework.

Since it was implemented, the player numbers has increased, we have received more ways to get the power and the view range has increased slightly iirc. all of that, especially the amount of people just adds onto a power that was already very powerful to begin with.

**Comment by tigercat2000** [comment]
As Ansari said, this is not going to ever feasibly pass.
